### DISCUSSION TITLE
Process community rewards cycle: 9/24 -> 10/21

### DISCUSSION THREAD

**Discussion by AlgoRhythMatic** [asastats/channel#301] [Rewards]
Another cycle has passed, so it is once again time to distribute rewards to our community contributors based on measurements listed in the bottom of the page of our [Ongoing Contributions spreadsheet](https://docs.google.com/spreadsheets/d/1UJ9ZyvV0fdcXAfpUNOsRa1aCmJko-QhF-BU4Wcu6Xog/edit#gid=238932857):

```
# NOTE ASA Stats Community rewards 9/24 - 10/21
@fredestante 0.06 damo
AlgoRhythMatic 0.36 damo
BartleDooDerp.algo 0.15 damo
Damo 0.21 damo
DragMZ 0.09 damo
Maru the Cat 0.15 damo
moloch10 0.09 damo
petled 0.03 damo
prince2486 0.03 damo
SCN9A 0.36 damo
u/Ansuz4u 0.06 damo
u/Aromatic-Ad3922 0.03 damo
UnderSiege 0.06 damo
TOTAL 1.68 damo
```
Please review and comment if there are any questions or concerns. This period uses a 3x coefficient to multiply measurements from the [Rewards Model tab](https://docs.google.com/spreadsheets/d/1UJ9ZyvV0fdcXAfpUNOsRa1aCmJko-QhF-BU4Wcu6Xog/edit#gid=1130085233) from our spreadsheet.

Per usual process, we'll wait ~24 hrs for comments/feedback, and then pass on the list to the rewards sender (currently @ipaleka).

As well, please comment or reach out to me if you are interested in participating in rewards measurements in future cycles!

**Comment by AlgoRhythMatic** [comment]
Please note: this rewards cycle does not yet include rewards for all the valuable mobile application testing that has been underway.  Rewards for this work will be included in the same cycle that includes the release of the apps (likely as a cumulative one-time reward per participant), and then future rewards will be based on our traditional Github issue tracking method.

**Comment by ipaleka** [comment]
DragMZ [added three addresses](https://discord.com/channels/906917846754418770/973245225680109588/1031893770809122837) to create an NFT collection from them. Dunno how we should deal with such a request. We can probably add one F1 (or F2) for DragMZ in this cycle.

**Reply by ipaleka** [reply]
Also, I guess we can add to this cycle the following issue that references original moloch10 request: https://github.com/asastats/channel/issues/300

**Reply by AlgoRhythMatic** [reply]
> DragMZ [added three addresses](https://discord.com/channels/906917846754418770/973245225680109588/1031893770809122837) to create an NFT collection from them. Dunno how we should deal with such a request. We can probably add one F1 (or F2) for DragMZ in this cycle.

Added as an F2.

**Reply by AlgoRhythMatic** [reply]
Adjusted list w/ the two updates noted above:
```
# NOTE ASA Stats Community rewards 9/24 - 10/21
@fredestante 0.06 damo
AlgoRhythMatic 0.36 damo
BartleDooDerp.algo 0.15 damo
Damo 0.21 damo
DragMZ 0.15 damo
Maru the Cat 0.15 damo
moloch10 0.12 damo
petled 0.03 damo
prince2486 0.03 damo
SCN9A 0.36 damo
u/Ansuz4u 0.06 damo
u/Aromatic-Ad3922 0.03 damo
UnderSiege 0.06 damo
TOTAL 1.77 damo
```
